eng/pipelines/coreclr/templates/build-job.yml

```diff
@@ -210,7 +216,7 @@ jobs:
         displayName: Run CoreCLR Tools unit tests
 
     # Build native test components
-    - ${{ if ne(parameters.isOfficialBuild, true) }}:
+    - ${{ if and(ne(parameters.isOfficialBuild, true), ne(parameters.archType, 'armel'))}}:
```

Comment:
Could it be that it was failing native test subset just because `-portablebuild=false` is not passed?

Reply:
Right. It needs a portable option and some others.
However, as I know, there is no ARMEL environment for test in the CI.
To prevent build failure, I just want to add Tizen ARMEL build this time.
Thank you.